doc/03_reference/coverage_plan.rst

```diff
@@ -229,8 +228,10 @@ PMP
 * Access fail & pass.
 
   * All combinations of unaligned access split across a boundary, both halves pass, neither pass, just the first passes, just the second passes.
+
     * Two possible boundary splits; across a 32-bit boundary within a region or a boundary between PMP regions.
-  * ``cp_pmp_iside_region_override`` /  ``cp_pmp_dside_region_override`` - Higher priority entry allows access that lower priority entry prevents.
+
+  * ``cp_pmp_iside_region_override``, ``cp_pmp_dside_region_override`` - Higher priority entry allows access that lower priority entry prevents.
```

Comment:
Would it makes sense to cover the second i-side check here separately?

Reply:
Yes defintely, this was originally written before we added the second iside channel and I failed to update this bit.

Reply:
In fact I'd already implemented this, just failed to document it.